packages/performance/.eslintrc.cjs

```diff
@@ -1,18 +1,6 @@
 module.exports = {
     root: true,
-    extends: [
-        '@ni/eslint-config-javascript'
-    ],
-    ignorePatterns: [
-        // Force inclusion of config dot file
-        '!.eslintrc.js',
-        'node_modules',
-        'dist'
-    ],
-    rules: {
-        // Enabled to prevent accidental usage of async-await
-        'require-await': 'error'
-    },
+    extends: 'plugin:@ni-private/nimble/javascript',
     overrides: [{
         files: ['*.ts'],
```

Comment:
Could you share the thought process around sharing the JS part of this config but not the TS part?

Reply:
I don't think this file should be updated. The performance package is not authoring nimble components iin FAST so the normal '@ni/eslint-config-javascript' was sufficient.